dbt/adapters/bigquery/connections.py

```diff
@@ -239,6 +240,10 @@ class BigQueryConnectionManager(BaseConnectionManager):
     DEFAULT_INITIAL_DELAY = 1.0  # Seconds
     DEFAULT_MAXIMUM_DELAY = 3.0  # Seconds
 
+    def __init__(self, profile: AdapterRequiredConfig):
+        super().__init__(profile)
+        self.jobs_by_thread: Dict[Any, Any] = {}
```

Comment:
do we have any more stricter sense we want here should this just be str?

Reply:
I'm pretty sure this can just be `str, str`